packages/in_app_purchase/in_app_purchase_storekit/example/shared/RunnerTests/ProductRequestHandlerTests.swift

```diff
@@ -0,0 +1,91 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import StoreKit
+import XCTest
+
+@testable import in_app_purchase_storekit
+
+class RequestHandlerTest: XCTestCase {
+
+  func testRequestHandlerWithProductRequestSuccess() {
+    let productIdentifiers = Set(["123"])
+    let request = SKProductRequestStub(productIdentifiers: productIdentifiers)
+    let handler = FIAPRequestHandler(request: request)
+
+    let expectation = expectation(description: "Expect response with 1 product")
+    var response: SKProductsResponse?
+
+    handler.startProductRequest { r, error in
```

Comment:
nit: 
(1) do not use single letter `r` for variable name
(2) unused `error`. either use it or use `_`